test/utils/describeConformanceV5.js

```diff
@@ -242,6 +242,35 @@ const fullSuite = {
   themeVariants: testThemeVariants,
 };
 
+export function validateOptions(options, testKeys = []) {
+  const checklist = [
```

Comment:
Actually, I wonder. Why not make these two options required for the test suite? I mean. It feels that we could turn this PR into a +-5 LOCs. We could throw if render or muiName are missing

Reply:
😂 updated, looks simpler